packages/core/src/auth/auth.ts

```diff
@@ -946,24 +946,13 @@ export class Auth implements AuthService, ConnectionManager {
 
     // Used by Amazon Q to delete connection status & scope when this deletion is made by AWS Toolkit
```

Comment:
Ideally, any Q-only logic would be in a Q-only module. Not sure if that's possible currently.